splitio/sync/segment.py

```diff
@@ -145,26 +145,45 @@ def synchronize_segment(self, segment_name, till=None):
         attempts = _ON_DEMAND_FETCH_BACKOFF_MAX_RETRIES - remaining_attempts
         if successful_sync:  # succedeed sync
             _LOGGER.debug('Refresh completed in %d attempts.', attempts)
-            return
+            return True
```

Comment:
can we add  `:returns:` and `:rtype:` to docstring documenting what this method returns and it's type?